generateDapp.ts

```diff
@@ -23,40 +23,50 @@ export const generateDapp = async (opts: {
   network: Network;
   packageManager: PackageManager;
 }) => {
-  const repoName = opts[ARGUMENT_NAMES.NAME] || "my-aptos-dapp";
-  let repoAddr;
+  const projectName = opts[ARGUMENT_NAMES.NAME] || "my-aptos-dapp";
+  let templateDirectory;
   switch (opts[ARGUMENT_NAMES.TEMPLATE]) {
     case "todolist":
-      repoAddr = "https://github.com/aptos-labs/cad-todolist.git";
+      templateDirectory = "todolist";
       break;
-    case "new":
-      repoAddr = "https://github.com/aptos-labs/cad-boilerplate.git";
+    case "dapp-boilerplate":
+      templateDirectory = "dapp-boilerplate";
+      break;
+    case "node-boilerplate":
+      templateDirectory = "node-boilerplate";
       break;
     default:
-      repoAddr = "https://github.com/aptos-labs/cad-boilerplate.git";
+      templateDirectory = "dapp-boilerplate";
   }
-  const gitCheckoutCommand = `git clone ${repoAddr} ${repoName}`;
-  const deleteDotGitCommand = `rm -rf ${repoName}/.git`;
-  const installRootDepsCommand = `cd ${repoName} && ${opts.packageManager} install`;
-  const replaceNpmUsagesCommand = `cd ${repoName} && sed -i.bak 's/npm/${opts.packageManager}/g' package.json && rm package.json.bak`;
+  const copyTemplateToProjectDirectory = `cp -r templates/${templateDirectory} ${projectName}/`;
+  const installRootDepsCommand = `cd ${projectName} && ${opts.packageManager} install`;
+  const replaceNpmUsagesCommand = `cd ${projectName} && sed -i.bak 's/npm/${opts.packageManager}/g' package.json && rm package.json.bak`;
 
-  // Clone the repo
-  console.log("Cloning template repo...");
-  const checkedOut = runCommand(gitCheckoutCommand);
-  const deleteDotGit = runCommand(deleteDotGitCommand);
-  if (!checkedOut || !deleteDotGit) {
-    console.error("Failed to clone the repo");
+  // Creating project directory
+  console.log("Creating project...");
+  const copiedTemplate = runCommand(copyTemplateToProjectDirectory);
+  if (!copiedTemplate) {
+    console.error("Failed to create project directory");
     process.exit(-1);
   } else {
-    console.log("Cloned successfully");
+    console.log("Created successfully");
   }
 
   // create .env file
   console.log("Creating .env file...");
-  runCommand(`cd ${repoName} && touch .env`);
+  runCommand(`cd ${projectName} && touch .env`);
   const network = opts[ARGUMENT_NAMES.NETWORK] || "testnet";
-  runCommand(`echo "VITE_APP_NETWORK=${network}" > ${repoName}/.env`);
-  runCommand(`echo "VITE_APP_NETWORK=${network}" > ${repoName}/frontend/.env`);
+
+  // TODO more sophisticate way to distinguish between node and web env
+  if (templateDirectory === "node-boilerplate") {
+    runCommand(`echo "APP_NETWORK=${network}" > ${projectName}/.env`);
```

Comment:
How come we have APP_NETWORK and VITE_APP_NETWORK?

Reply:
vite requires env variables to start with `VITE_` (same as react), we use no `vite` for a node dapp (that doesnt use vite)